doc/nrf/ug_matter_device_certification.rst

```diff
@@ -110,6 +110,19 @@ The final application for certification requires you to send the following docum
 * `Matter Attestation of Security`_
 * Description of the Matter component to be certified (and photos of the end product, for the product certification)
 
+Some of the certification documents require providing information about Nordic Semiconductor's SoCs or the |NCS|.
+To help you with filling in these documents, Nordic Semiconductor can provide you with appropriate templates for:
+
+* Matter Attestation of Security
+* Thread certification by inheritance application
+
+To request and obtain such documents, open a private ticket on `DevZone`_ after you join the related SDO.
+
+.. note::
+   Nordic Semiconductor provides just an example answers based on the reference SDK applications, which may guide you on how to answer to certain platform-related questions.
+   You are still solely responsibility for filling original documents according to a product specifiction.
+   The final approval depends on the Director of Certification in the given Standards Developing Organization (SDO).
+
```

Comment:
I have a problem with this edit.
The page was written with the following structure in mind: 1) general description of the certification process (platform-agnostic), 2) certification scenarios, 3) inheritance scenarios, 3a) inheritance using Nordic's Matter solution.

Moving 3a) under 1) breaks the whole flow and (being picky here!) can give the idea that info from Nordic is needed for each and every certification application for Matter and that we can help all customers in the whole world ;)

Can we just link to the last section from under the list of required docs?

```
.. note::
   If you are applying for certification of a product that uses one of Nordic Semiconductor's stacks, you can ask for help with filling the certification documents. See `Certification document templates`_ for details.
```

Reply:
It's fine with me to make a separate section and link to it from here.

Note however, that Security Attestation document is needed for every certification application - so that might be useful.